src/coreComponents/physicsSolvers/fluidFlow/FlowSolverBase.cpp

```diff
@@ -707,9 +707,9 @@ void FlowSolverBase::saveAquiferConvergedState( real64 const & time,
 
     if( bc.getLogLevel() >= 1 )
     {
-      GEOS_LOG_RANK_0( GEOS_FMT( string( "FlowSolverBase {}: at time {}s, " )
-                                 + string( "the <{}> boundary condition '{}' produces a flux of {} kg (or moles if useMass=0). " ),
-                                 getDataContext(), time+dt, AquiferBoundaryCondition::catalogName(), bc.getName(), dt * globalSumFluxes[aquiferIndex] ) );
+      GEOS_LOG_RANK_0( GEOS_FMT( "{} {}: at time {}s, the boundary condition produces a flux of {} m3.",
+                                 AquiferBoundaryCondition::catalogName(), bc.getDataContext(),
```

Comment:
I am not a big fan of so detailed output about xml and line here to be honest.
I think it makes total sense for input error messages but not so much value for simulation dynamic data such as aquifer flux here. Just aquifer name is enough imho.

Reply:
I insist on not printing xml file name and line number here

Reply:
What is the output that @MelReyCG 's PR will produce?

Reply:
and I would appreciate spaces between values and units, forgot about that before, sorry

Reply:
Hmm, yes it makes sense. If there's nothing that the user should be doing on line `226` to save fix the issue (if this even is an issue...), then we could remove the information to reduce the noise.
@paveltomin Are there any other places where you'd like this pattern to be changed?
Also, we're now top of the merge queue. Should we move on nevertheless?
What's your take @jeannepellerin ?

Reply:
Thanks, I think you got my point. For input errors reporting file name and location is very helpful. Doesn't make much sense for solver issues, and even less sense for results reporting.
I can check other places tomorrow, but I am also ok with merging, it's not that big deal.